docs/maintainers/beginner-issues-guidelines.md

```
@@ -0,0 +1,167 @@
+
+- Extending existing tests to cover specific additional scenarios
+- Adding assertions based on observed behavior
+- Improving test clarity, naming, or intent
+
+#### Not Allowed
+
+- Creating new unit or integration test suites
+- Designing complex or high-level test strategies
+
+---
+## What is Not a Beginner Issue
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)